scripts/generate_conditions.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+_condition_strings = [
+    "ArnEquals",
+    "ArnNotEquals",
+    "ArnLike",
+    "ArnNotLike",
+    "Bool",
+    "DateEquals",
+    "DateNotEquals",
+    "DateLessThan",
+    "DateLessThanEquals",
+    "DateGreaterThan",
+    "DateGreaterThanEquals",
+    "IpAddress",
+    "NotIpAddress",
+    "Null",
+    "NumericEquals",
+    "NumericNotEquals",
+    "NumericLessThan",
+    "NumericLessThanEquals",
+    "NumericGreaterThan",
+    "NumericGreaterThanEquals",
+    "StringEquals",
+    "StringNotEquals",
+    "StringEqualsIgnoreCase",
+    "StringNotEqualsIgnoreCase",
+    "StringLike",
+    "StringNotLike",
+]
+
+_condition_qualifier_strings = ["ForAnyValue", "ForAllValues"]
+
+
+def make_condition(type_name: str, condition_name: str) -> None:
+    print(f"class {type_name}(ConditionElement):")
+    print(f"    condition = '{condition_name}'")
```

Comment:
This should also be accepted by black but produces code that doesn't need formatting by black.  Same in line 41.

```suggestion
    print(f'    condition = "{condition_name}"')
```